taglib/toolkit/tfile.cpp

```diff
@@ -149,6 +151,10 @@ PropertyMap File::properties() const
     return dynamic_cast<const MP4::File* >(this)->properties();
   if(dynamic_cast<const ASF::File* >(this))
     return dynamic_cast<const ASF::File* >(this)->properties();
+  if (dynamic_cast<const DSF::File* >(this))
+    return dynamic_cast<const DSF::File* >(this)->properties();
+  if (dynamic_cast<const DSDIFF::File* >(this))
```

Comment:
I didn't even look at existing TagLib files modified with this PR.
Shame on me!

Thus these  two places have a space between function name and parenthesis ...